pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.9.8
+title: Release Notes for 1.9.8
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.8.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.8/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.8
+
+- COPS-1879 - Fixed the failure to kill a docker task in staging.
+- CORE-1395 - Renamed mesos-modules-private to dcos-ee-mesos-modules.
+- DCOS-20631 - dcos-vault: Fixed deadlock in go-zookeeper library (1.9).
+- DCOS-20632 - dcos-secrets: Fixed deadlock in go-zookeeper library (1.9).
+- DCOS-20754 - Included sec-cli-lib `user_is_authorized` function in all #sec CIs.
+- DCOS-21002 - Fixed release Marathon Plugin by adding full authorization to resources on DCOS 1.9.
+- DCOS-21451 - Prevented reusing tcp sockets by AR's cache code [master] in admin router.
```

Comment:
We should reword that so that it's clear what we've done here. It's a beautiful change, and the current sentence does not communicate the value.

Reply:
Proposal:

`DCOS-21451 - Fixed a bug as of which Admin Router would not pick up Mesos leader changes (leading to unexpected 404 HTTP responses when using the service endpoint)`

Reply:
CC @vespian

Reply:
Reworded to:
DCOS-21451 - Fixed a bug where the Admin Router would not pick up Mesos leader changes (leading to unexpected 404 HTTP responses when using the service endpoint).